app/controllers/api/containers_controller.rb

```diff
@@ -5,24 +5,32 @@ def index
 
   def create
     container = Container.new(container_params)
-    container.image = Image.find_by_image_id(params[:image_id])
-    container.save
-    render json: container
+    container.image = Image.find(params[:container][:image_id])
+    if container.save
+      render status: 200, json:{container: container}
+    else
+      render status: 500, json: {error: container.errors}
+    end
   end
 
   def delete
     container = Container.find(params[:id])
     if !container.nil?
-      container.destroy!
+      container.destroy
+      if container.destroyed?
```

Comment:
Nope, container.destroy returns a frozen object (see in source code here http://apidock.com/rails/ActiveRecord/Base/destroy)